engine/access/rpc/backend/backend_test.go

```diff
@@ -897,22 +852,32 @@ func (suite *Suite) TestGetEventsForBlockIDs() {
 }
 
 func (suite *Suite) TestGetEventsForHeightRange() {
-	suite.state.On("Sealed").Return(suite.snapshot, nil).Maybe()
 
 	ctx := context.Background()
 	const minHeight uint64 = 5
 	const maxHeight uint64 = 10
 	var headHeight uint64
 	var blockHeaders []*flow.Header
+	var nodeIdentifiers flow.IdentityList
```

Comment:
```suggestion
	var nodeIdentities flow.IdentityList
```
the type is identity list, not identifier list